mvnw11

```diff
@@ -32,8 +28,6 @@ echo "> ./mvnw \"$@\" -P java11"
 ./mvnw "$@" -P java11
```

Comment:
For the sake of review, I'll pick on something you didn't even change.  Test failures in this call aren't propagated to the script result, so the `mvn test` can fail gloriously but this script will happily exit with no problems.